source/support.tex

```diff
@@ -1884,6 +1913,27 @@
 
 \xrefc{7.20}
 
+\pnum
+All types that use the placeholder \placeholder{N}
+are optional when \placeholder{N}
+is not \tcode{8}, \tcode{16}, \tcode{32}, or \tcode{64}.
+The exact-width types
+\tcode{int\placeholdernc{N}_t} and \tcode{uint\placeholdernc{N}_t}
+for \placeholder{N} = \tcode{8}, \tcode{16}, \tcode{32}, and \tcode{64}
+are also optional;
+however, if an implementation defines integer types
+with the corresponding width and no padding bits,
+it defines the corresponding typedef names.
+Each of the macros listed in this subclause
+is defined if and only if
+the implementation defines the corresponding typedef name.
+\begin{note}
+The macros \tcode{INT\placeholdernc{N}_C} and \tcode{UINT\placeholdernc{N}_C}
+correspond to the typedef names
```

Comment:
962402ca37d40fb748cce127f122a578c951b2af uses `\grammarterm{typedef-name}`. What's the preference?

Reply:
"typedef name" and the grammar term "typedef-name" might not be equivalent, so I'm not comfortable changing this without LWG's OK.  Please open an issue?

Reply:
Core wording seems to use them equivalently. If they're not equivalent, what is a "typedef name"? It's never defined.

Reply:
I'm with you 100% - "typedef name" should be defined. Care to open an issue? I would, except that I already annoy CWG to no end with my requests for precise definitions :)

Reply:
As @zygoloid explained on the reflector, we should rename this. Let's do that separately, #4401.